.nvmrc

```diff
@@ -1 +1 @@
-v18
\ No newline at end of file
+18
```

Comment:
I'm not sure about this change 🤔 
Some inet research all suggest to use something like `node -v > .nvmrc`
And `node -v` prints out `v18.xxx`